btr-web/btr-main-app/services/file-significant-individual.ts

```diff
@@ -117,9 +117,17 @@ const getPersonAndOwnershipAndControlStatements = (sif: SignificantIndividualFil
 
 const convertToBtrBodsForSubmit = (sif: SignificantIndividualFilingI): BtrFilingI => {
   const businessDetails = getCurrentBusinessAsBtrBodsEntityI()
-
   const { ownershipOrControlStatements, personStatements } = getPersonAndOwnershipAndControlStatements(sif)
 
+  for (const personS of personStatements) {
```

Comment:
Is this fix for some UI situation ?
Why are we deleting keys from objects that are empty ?

Reply:
This is for the api actually, because if the UI passes along '' many of the keys fail

Reply:
I think it would be good to add a comment here